Anseys/graph.gv

```diff
@@ -5,14 +5,18 @@ digraph {
 
     O -> x[dir=none, style="dashed"];
```

Comment:
Sorry to be so annoying, but I believe we decided that «dashed» was to be used for contaminations. Here, the dash line in the original stemma has a different meaning (showing that the archetype is not the original).

Reply:
You're not annoying at all. I changed it in the push